picard/tagger.py

```diff
@@ -1047,8 +1052,9 @@ def process_picard_args():
     parser.add_argument("-V", "--long-version", action='store_true',
                         help="display long version information and exit")
     parser.add_argument('FILE', nargs='*')
-    picard_args, unparsed_args = parser.parse_known_args()
-    return picard_args, unparsed_args
+
+    # return only picard args that were actually parsed, in index [1] there are stored unparsed args
+    return parser.parse_known_args()[0]
```

Comment:
Why was this change needed?

Reply:
It's not needed for pipes to work but I thought it's a fine opportunity to refactor the `Tagger` class (especially its arguments structure and the constructor) and the start process in general.

`unparsed_args` were used only in `Tagger` constructor (but were unused in the class), so I think such change makes the code more clean

Reply:
We usually try to keep such unrelated changes outside feature PRs, so for now I'd prefer those changes to be left out.

Reply:
Okay, I've undone my changes in the new commit.